components/script/dom/window.rs

```diff
@@ -573,6 +573,20 @@ impl WindowMethods for Window {
         self.window_proxy().open(url, target, features)
     }
 
+    #[allow(unsafe_code)]
+    // https://html.spec.whatwg.org/multipage/#dom-opener
+    unsafe fn Opener(&self, cx: *mut JSContext) -> JSVal {
+        self.window_proxy().opener(cx)
+    }
+
+    #[allow(unsafe_code)]
+    // https://html.spec.whatwg.org/multipage/#dom-opener
+    unsafe fn SetOpener(&self, _cx: *mut JSContext, _value: HandleValue) {
+        // TODO: Step 2.
+        // NOTE: for now assuming value is null, and disowning.
```

Comment:
This seems like an important todo to me, shouldn't that land at the same time as the rest of the PR?

Reply:
Ok, I've made an attempt